### PR TITLE
Better German lang

### PR DESCRIPTION
**Pull request by robske110**
@thebigsmileXD correct me if i did something wrong

### CODE REVIEW

**Comment by robske110** [comment]
Is this going to be merged?
(It was even created before i got an team member lol)

File: src/pocketmine/lang/locale/deu.ini [hunk 1]
```diff
@@ -7,7 +7,7 @@ multiplayer.player.left={%0} hat das Spiel verlassen
 
 chat.type.achievement={%0} hat die Errungenschaft {%1} erzielt
 
-disconnectionScreen.outdatedClient=Veralteter Client!
+disconnectionScreen.outdatedClient=Veraltetes Minecraft! #Almost nobody in Germany knows what a Client is!
```
Comment (inxomnyaa) [inline src/pocketmine/lang/locale/deu.ini:10]: This is wrong. Its.. KID LANGUAGE

File: src/pocketmine/lang/locale/deu.ini [hunk 2]
```diff
@@ -228,7 +228,7 @@ pocketmine.command.reload.reloaded=Neuladen beendet.
 pocketmine.command.status.description=Gibt die Serverauslastung aus.
 pocketmine.command.status.usage=/status
 
-pocketmine.command.gc.description=Startet Automatische Speicherbereinigungs Aufgaben
+pocketmine.command.gc.description=Startet Automatische Speicherbereinigung
```
Comment (inxomnyaa) [inline src/pocketmine/lang/locale/deu.ini:231]: automatische klein

File: src/pocketmine/lang/locale/deu.ini [hunk 3]
```diff
@@ -284,7 +284,7 @@ pocketmine.command.spawnpoint.description=Setzt den Spawnpunkt eines Spielers
 pocketmine.command.stop.description=Stoppt den Server
 pocketmine.command.tp.description=Teleportiert den gegebenen Spieler (oder einen selbst) zu einem anderen Spieler oder zu Koordinaten
 pocketmine.command.tell.description=Sendet eine private Nachricht an den angegebenen Spieler
-pocketmine.command.whitelist.description=Managt die Liste der Spieler, denen es erlaubt ist den Server zu benutzen
+pocketmine.command.whitelist.description=Verwaltet die Liste der Spieler, denen es erlaubt ist den Server zu benutzen
```
Comment (inxomnyaa) [inline src/pocketmine/lang/locale/deu.ini:287]: erlaubt ist, den

File: src/pocketmine/lang/locale/deu.ini (hunk 2 above)
Comment (SalmonDE) [inline src/pocketmine/lang/locale/deu.ini:231]: Vielleicht könnte man auch noch ein "die" hinzufügen. Also "Startet die automatische Speicherbereinigung"

**Comment by ghost** [comment]
asfasfhs? hehe idk what yo sain'

**Comment by robske110** [comment]
@thebigsmileXD @SalmonGER Fixed all your suggestions here now: https://github.com/ClearSkyTeam/ClearSky/pull/144/files
Still need help with the @TODO stuff at the end of the file deu.ini

**Comment by SalmonDE** [comment]
@robske110 Wow, thanks. That will help me a lot if this PR gets merged!

**Pull request merged**